posthog/models/instance_setting.py

```diff
@@ -22,8 +23,14 @@ def value(self):
 def get_instance_setting(key: str) -> Any:
     assert key in CONSTANCE_CONFIG, f"Unknown dynamic setting: {repr(key)}"
 
+    cached_setting = cache.get(key)
```

Comment:
Do we avoid thundering herd here? I recall there being a specific Django cache client that handles this, not sure if we're using it. And do we want to protect against this?

Reply:
I think we are using the default Redis client for cache. I don't see anything related to handling load.

Reply:
Looks like we are rolling with the regular client and not the herd client. The API for both is very similar though so I'm testing out the herd client and seeing if we can make the swap safely.